components/Card.js

```diff
@@ -17,11 +17,13 @@ const StyledListElement = styled.li`
   list-style-type: none;
 `;
 
-export default function Card() {
+export default function Card({ location }) {
   return (
     <StyledListElement>
       <StyledLink href="#">
-        <CardElement>Name of Location</CardElement>
+        <CardElement h>
```

Comment:
is the "h" a prop from styled components or a typo?

Reply:
Just a typo, removed it, thanks!